components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/templates/application/applicationKeys.hbs

```diff
@@ -0,0 +1,187 @@
+{{#*inline "callback_url" }}
+    <div class="form-group">
+        <label class="" for="ConsumerKey">Callback URL</label>
+        <input id="callbackUrl" type="text" class="form-control callback_url">
+    </div>
+{{/inline}}
+
+{{#*inline "grants" }}
+    <div class="row add-margin-top-5x">
+        <div class="col-md-12">
+            <h4>Grant Types</h4>
+            <p>Application can use the following grant types to generate Access Tokens. Based on the application requirement, you can enable or disable grant types for this application.</p>
```

Comment:
Add multi language support. Check the full PR and add them. There are multiple places to add it.